### DISCUSSION TITLE
Force <LightMode /> don't work on <Text />

### DISCUSSION THREAD

**Discussion by tannguyencse19** [chakra-ui/chakra-ui#5273] [Q&A] [answered]
I am forcing `<LightMode />` on `<Text />` and `<Button />` components. Button works fine while Text doesn't. 

```jsx
      <LightMode>
        <VStack alignItems="start">
          <Text fontSize="3xl">60% Discount</Text>
          <Text fontSize="8xl" fontWeight="bold">
            Winter <br /> Collection
          </Text>
          <Text>Best Cloth Collection By 2020</Text>
          <Button colorScheme="messenger" borderRadius="3xl" size="lg">
            Shop Now
          </Button>
        </VStack>
      </LightMode>
```

![LightModeText](https://user-images.githubusercontent.com/62830718/146967187-5ed6be0b-cf25-4e96-a10b-6c70a03fa4e6.gif)

I have try on different code places but other components work fine expect Text. 😕

**Comment by primos63** [comment, marked as answer]
Yes because `Text` has no theme styling, although you can add one, so it falls under the global styles that are applied
```javascript
const styles: Styles = {
  global: (props) => ({
    body: {
      fontFamily: "body",
      color: mode("gray.800", "whiteAlpha.900")(props), 
      bg: mode("white", "gray.800")(props),
      transitionProperty: "background-color",
      transitionDuration: "normal",
      lineHeight: "base",
    },
    "*::placeholder": {
      color: mode("gray.400", "whiteAlpha.400")(props),
    },
    "*, *::before, &::after": {
      borderColor: mode("gray.200", "whiteAlpha.300")(props),
      wordWrap: "break-word",
    },
  }),
}
```
Here's a [CodeSandbox](https://codesandbox.io/s/lightmode-text-hrxjc) example of adding a color style to `Text`. It now obeys the `LightMode` component.

**Reply by nvmnghia** [reply]
Can you add a list of components that can be forced with `<LightMode>` without modification?
